### PR TITLE
CVE-2005-3352

### PR DESCRIPTION
**Pull request by sxm7571**

### CODE REVIEW

**Comment by rburchlau001** [comment]
What is going on with this one? Are the "test" entries just an attempt to get the .yml working?

**Comment by sxm7571** [comment]
> What is going on with this one? Are the "test" entries just an attempt to get the .yml working?

Yeah I can't get this one to build properly.

**Comment by rburchlau001** [comment]
> > What is going on with this one? Are the "test" entries just an attempt to get the .yml working?
> 
> Yeah I can't get this one to build properly.

I'll try to take a look and help out this weekend if you don't get it by then.

**Comment by sxm7571** [comment]
> > > What is going on with this one? Are the "test" entries just an attempt to get the .yml working?
> > 
> > 
> > Yeah I can't get this one to build properly.
> 
> I'll try to take a look and help out this weekend if you don't get it by then.

Okay, thank you.

File: cves/CVE-2005-3352.yml [hunk 1]
```diff
@@ -55,10 +59,8 @@ fixes:
    - commit: 9de8a693d16e5ce76ff5b7f727311e783a51f34a
      note:
 vccs:
-  - commit: 9de8a693d16e5ce76ff5b7f727311e783a51f34a
-    note: Rename from mod_imap
-  - commit: a83b21763cd2fbfd97bf5cbb5f01ea4d7e7ac3c3
-    note:
+   - commit: Patch for apache-1.3 tree (ack fielding, jorton)/apache-2.0 tree (ack fielding, jorton) - Can't find commit ID
```
Comment (rburchlau001) [inline cves/CVE-2005-3352.yml]: this might be failing the build, should throw this in the note and leave commit blank

Comment (sxm7571) [reply cves/CVE-2005-3352.yml]: I agree with that, you can only put a commit ID there.

Comment (sxm7571) [reply cves/CVE-2005-3352.yml]: I just removed the indentations and the build passed. I appreciate the help.

**Pull request merged**